src/screens/ScanQrAddressScreen.tsx

```diff
@@ -0,0 +1,81 @@
+/* global alert */
```

Comment:
Couldn't that be resolved using `ScanQrCodeScreen` which I've created for scanning address in Address book? The code is really similar and there's also possibility to pass the function.